packages/client/src/features/tools/change-bounds/change-bounds-tool.ts

```diff
@@ -63,6 +63,10 @@ import {
 import { FeedbackMoveMouseListener } from './change-bounds-tool-move-feedback';
 import { ChangeBoundsTracker, TrackedElementResize, TrackedResize } from './change-bounds-tracker';
 
+export interface IMovementBehavior {
```

Comment:
I'm a little bit on the fence about whether it should be called `IMovementBehavior` (as we have with the `IMarqueeBehavior`) or `IMovementOptions` (as we have with the `IHelperLineOptions`). I think both are probably fine, just wanted to note that down somewhere. We should probably unify that at some point.

Reply:
I named it indeed after the MarqueeBehavior (as I invented this too 🙈), however, I think I like Options more, so I will call it options.